imageroot/systemd/nextsec-controller-notifier.service

```
@@ -0,0 +1,8 @@
+[Unit]
+Description=Add client as service provider for prometheus
+BindsTo=MODULE_ID.service
+After=MODULE_ID-vpn.service
+
+[Service]
+ExecStart=/usr/local/bin/runagent node-exporter-handler /var/lib/nethserver/MODULE_ID/state/clients
```

:warning:  the script name has `_` not `-`, choose the one you prefer but this line should be change for sure:
```suggestion
ExecStart=/usr/local/bin/runagent metrics-exporter-handler /var/lib/nethserver/MODULE_ID/state/clients
```